Quadriga/src/main/java/edu/asu/spring/quadriga/service/projectblog/IProjectBlogEntryManager.java

```diff
@@ -38,4 +38,14 @@
      * @throws QuadrigaStorageException
      */
     List<IProjectBlogEntry> getProjectBlogEntryList(String projectId, Integer count) throws QuadrigaStorageException;
+    
+    /**
+     * 
+     * @param projectBlogEntryId
+     *            blog entry id used to obtain the blog entry   
+     * @return  project blog entry
+     * @throws QuadrigaStorageException
+     */
+    IProjectBlogEntry getProjectBlogEntryDetails(String projectBlogEntryId) throws QuadrigaStorageException ;
```

Comment:
What does this method do? Description is missing. Also 'blog entry id used to obtain the blog entry' is a little inaccurate. The method gets the id of the blog entry that should be retrieved from the database.